data-dashboard/.eslintrc.json

```diff
@@ -7,14 +10,44 @@
     "plugin:@typescript-eslint/recommended",
     "airbnb",
     "airbnb/hooks",
-    "plugin:@typescript-eslint/recommended"
+    "plugin:@typescript-eslint/recommended",
+    "plugin:import/typescript"
   ],
   "globals": {
     "JSX": true
   },
   "settings": {
     "react": {
       "version": "detect"
+    },
```

Comment:
Could you please remove these changes? I don't think changing our eslint is necessary